src/scripts/tests/import-open-api.test.ts

```diff
@@ -60,7 +60,7 @@ describe("scripts/import-open-api", () => {
       { item: { type: "array", items: { type: "integer" } }, expected: "number[]" },
       { item: { type: "array", items: { type: "customType" } }, expected: "any[]" },
       { item: { type: "object", properties: { value: { type: "integer" } } }, expected: "{value?: number}" },
-      { item: { type: "object" }, expected: "{}" },
+      { item: { type: "object" }, expected: "{[key: string]: any}" },
```

Comment:
I also improve a bit this type, the `additionalProperties` is implicit in this part if I understand correctly the specs, so it's not an empty object.